### PR TITLE
Bump libp2p deps for uint8arrays@3.0.0

### PR DESCRIPTION
**Pull request by dapplion**
Ensure the tree contains no uint8arrays version under 3.0.0

CC https://github.com/libp2p/js-libp2p/issues/967

### CODE REVIEW

**Comment by dapplion** [comment]
Installed from local file and no uint8arrays < v3 is installed :+1: 

```
yarn init -y
yarn add @chainsafe/lodestar@file:../../lodestar/packages/lodestar/
```

**Comment by dapplion** [comment]
Blocked by https://github.com/libp2p/js-libp2p-tcp/issues/149

**Review by twoeths — APPROVED** [review]
we need to try using docker again after merging this 👍

**Pull request merged**
